blazor/dropdown-list/multicolumn.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: Multicolumn in Blazor DropDown List Component | Syncfusion
+description: Checkout and learn here all about Multicolumn in Syncfusion Blazor DropDown List component and much more.

```

Comment:
Description: Checkout and learn here all about Multicolumn in Syncfusion Blazor DropDown List component and more.